openlibrary/templates/recentchanges/merge/comment.html

```diff
@@ -13,7 +13,7 @@
         $ details = ''
     $else:
         $ context = page.key
-        $ details = $:_('See <a href="' + change.url() + '">details</a>.')
+        $ details = _('See <a href="%s">details</a>.' % change.url())
```

Comment:
```suggestion
        $ details = _('See <a href="%s">details</a>.', change.url())
```
This may fix the error, and I think we may also need the `$:` because of where on the page it's rendered, as in the `title` fields at the top of some of these files.